### ISSUE TITLE
Broken tag on homepage "Projects" navigation

### ISSUE THREAD

**Issue by Uninen** [beeware/beeware.github.io#379]
The homepage projects-menu looks like there's a broken Lektor tag:
```
<lektor_rst.rst object="" at<br="">0x7f4fe342df10&gt;
    </lektor_rst.rst>
```

Don't quite know what's happening here as I'm not familiar with the tool but output looks like this: 

<img width="324" alt="Screenshot 2020-02-11 at 21 56 32" src="https://user-images.githubusercontent.com/43100/74273684-876fe580-4d19-11ea-88ef-db5636b3da78.png">

**Comment by goanpeca** [comment]
Thanks for reporting! will add a fix for this :-)

**Comment by freakboy3742** [comment]
I found the problem - it *should* have been fixed by #380... but now something is breaking when I try to publish the site :-(

**Comment by freakboy3742** [comment]
Ok - should now be fixed. Thanks for the report!
